docs/newsletter.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Newsletter
+title: Newsletter (old)
 parent: index
 id: Newsletter
 ---
 
+> Note: For the latest updated please check the [changelog](#CHANGELOG)
+
```

Comment:
I've decided to deprecate this document, we already have a changelog and will make a 8.0 upgrade guide. But if someone feels strongly that we should be releasing updates here too, then lets discuss it.

Reply:
the changes after this are from the autoformatter.